application/move_and_die.py

```diff
@@ -1,4 +1,4 @@
-from constans.constants_scores import CORRECT_MOVE
+from constans.constans import CORRECT_MOVE
```

Comment:
Did you mean 'constants" instead of 'constans'?

Reply:
Working on that now.